Chummer/Backend/Equipment/VehicleMod.cs

```diff
@@ -156,6 +157,10 @@ public void Create(XmlNode objXmlMod, int intRating, Vehicle objParent, decimal
             if (!objXmlMod.TryGetMultiLineStringFieldQuickly("altnotes", ref _strNotes))
                 objXmlMod.TryGetMultiLineStringFieldQuickly("notes", ref _strNotes);
 
+            String sNotesColor = ColorTranslator.ToHtml(ColorManager.HasNotesColor);
+            objXmlMod.TryGetStringFieldQuickly("notesColor", ref sNotesColor);
+            _colNotes = ColorTranslator.FromHtml(sNotesColor);
+
```

Comment:
Is this necessary in the `Create` method? Would there need to be data-definable colors for specific vehicle mods?